### PR TITLE
Nept 1584: Force variable xmlsitemap_prefetch_aliases to 0.

### PR DESCRIPTION
**Pull request by Fefaine**
### Description

Force variable xmlsitemap_prefetch_aliases to 0. This functionality has a very negative impact on performances.

### Change log

- Changed: value of variable xmlsitemap_prefetch_aliases set to 0.

### Commands

```sh
drush updb
```

### CODE REVIEW

File: tests/features/sitemap.feature [hunk 1]
```diff
@@ -4,11 +4,20 @@ Feature: Check the Sitemap
   I can publish a sitemap
 
   @api
+  Scenario: Value of the variable xmlsitemap_prefetch_aliases is zero
+    Given the module is enabled
+      | modules     |
+      | sitemap     |
+    Given I am an administrator user
```
Comment (Medi4tor) [inline tests/features/sitemap.feature]: Hi,
Maybe we can use 
Given I am logged in as a user with the "administrator" role

Comment (Fefaine) [reply tests/features/sitemap.feature]: Indeed, we have 2 different steps for this, we should only have one to be more consistent

**Comment by Medi4tor** [comment]
Seems OK for me, little changes mad in behat tests.

**Comment by aritomelo** [comment]
@Fefaine @Medi4tor I have checked the PR and I am wondering whether or not we should accept the second commit and the reason why is: I have tested both commits and both run without issue, so I do not know which one is the better.

**Comment by Fefaine** [comment]
@aritomelo  indeed both these steps are correct 
"Given I am an administrator user"
and
"Given I am logged in as a user with the "administrator" role"
But it is stupid to have two steps for the same action, so we will probably remove the first one. That's why i finally used the second one :)

**Comment by aritomelo** [comment]
All OK, ready to go!

**Pull request merged**